plugins/trendmicro_visionone/plugin.spec.yaml

```diff
@@ -0,0 +1,1629 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: trendmicro_visionone
+title: Trend Micro Vision One
+description: Trend Micro Vision One is a purpose-built threat defense platform that provides added value and new benefits beyond XDR solutions, allowing you to see more and respond faster. Providing deep and broad extended detection and response (XDR) capabilities that collect and automatically correlate data across multiple security layers, email, endpoints, servers, cloud workloads, and networks. Trend Micro Vision One prevents the majority of attacks with automated protection
+version: 1.0.0
+supported_versions: ["Trend Micro Vision One API v3"]
+vendor: trendmicro
+support: community
+status: []
+enable_cache: true
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/trendmicro_visionone
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://trendmicro.com
+tags:
+- vulnerability_scanning
+- vulnerability_management
+- intrusion_detection
+- intrusion_protection
+- virtual_patching
+- remediation
+- trend_micro
+- vision_one
+hub_tags:
+  use_cases: [vulnerability_management, cloud_security, remediation_management, threat_detection_and_response ]
+  keywords: [remediation, detection]
+  features: []
+
+types:
+  multi_response:
+    status:
+      title: Status
+      type: integer
+      description: Status Code of response
+      required: true
+    task_id:
+      title: Task ID
+      type: string
+      description: Task ID in Trend Micro Vision One of the executed action
+      required: false
+  suspicious_objects:
+    url:
+      title: URL
+      type: string
+      description: support leading and tailing wildcards
+      required: false
+    domain:
+      title: Domain
+      type: string
+      description: support leading wildcard
+      required: false
+    sender_mail_address:
+      title: Sender Mail Address
+      type: string
+      description: support fully qualified email address
+      required: false
+    ip:
+      title: IP
+      type: string
+      description: support only full match
```

Comment:
Ensure all descriptions start with a capital letter and do not end with punctuation.